### PR TITLE
MRG: Add automated helmet creation

### PR DESCRIPTION
**Pull request by larsoner**
While working on #5514 I realized it's useful to have a mechanism to approximate a MEG helmet on the fly. This PR adds that, and deletes the ARTEMIS123 file since we don't need it anymore (this creates an equivalent surface).

Also fixes a little bug with `dig=True` crashing for datasets without dig.

### CODE REVIEW

**Comment by larsoner** [comment]
https://10028-1301584-gh.circle-artifacts.com/0/html/auto_examples/visualization/plot_meg_sensors.html

File: mne/surface.py [hunk 1]
```diff
@@ -149,6 +170,16 @@ def get_meg_helmet_surf(info, trans=None, verbose=None):
     return surf
 
 
+def _reorder_ccw(rrs, tris):
+    """Reorder tris of a convex hull to be CCW-ordered."""
```
Comment (agramfort) [inline mne/surface.py]: CCW?

**Comment by agramfort** [comment]
KIT and magnes helmets are not great. Shoudl we switch to computed versions for these too?

**Comment by larsoner** [comment]
The computed ones also don't look great, just in a different way. So let's leave that for another PR

File: mne/surface.py (hunk 1 above)
Comment (larsoner) [reply mne/surface.py]: Counterclockwise, I'll clarify in code

**Comment by agramfort** [comment]
ok good. Feel free to merge when green.

**Comment by larsoner** [comment]
Failure is just spurious from Travis, will push comment update then merge (need this for the resting state PR)

**Pull request merged**